src/base/memory_monitor_sr.hpp

```diff
@@ -50,7 +50,7 @@ void mmon_add_stat (THREAD_ENTRY *thread_p, MMON_STAT_ID stat_id, int64_t size);
 void mmon_sub_stat (THREAD_ENTRY *thread_p, MMON_STAT_ID stat_id, int64_t size);
 void mmon_move_stat (THREAD_ENTRY *thread_p, MMON_STAT_ID src, MMON_STAT_ID dest, int64_t size);
 void mmon_resize_stat (THREAD_ENTRY *thread_p, MMON_STAT_ID stat_id, int64_t old_size, int64_t new_size);
-int mmon_aggregate_module_info (MMON_MODULE_INFO *info, int module_index);
-int mmon_aggregate_tran_info (MMON_TRAN_INFO *info, int tran_count);
+int mmon_aggregate_module_info (MMON_MODULE_INFO &info, int module_index);
+int mmon_aggregate_tran_info (MMON_TRAN_INFO &info, int tran_count);
```

Comment:
MMON_MODULE_INFO and MMON_TRAN_INFO have MMON_SERVER_INFO. It must be same information. How about extracting MMON_SERVER_INFO from them and make addtional API and protocol etc for it?